docs/api-usage/authentication.mdx

```diff
@@ -209,6 +209,11 @@ Follow a complete [video tutorial](https://www.youtube.com/watch?v=t6nxBk7JHCw)
 
 This is the most common way of authenticating users. It is used in the Saleor Storefront and Dashboard.
 
+:::caution
+If you are using a Single Sign-On provider, you will not be able to log in using email and password, 
```

Comment:
What If  my password is present in DB? 🤔 
My use case: I have an account created by Saleor, then I am able to log-in by SSO, and log in by normal flow. 
Thsi warning suggests me that the ☝️  is also not imposible, but AFAIR it was working 😅